webservices/tests/candidate_tests.py

```diff
@@ -96,22 +95,11 @@ def test_fields(self):
 
     def test_extra_fields(self):
         response = self._results('/candidate/P80003338?year=2008')
-        self.assertIn('committees', response[0])
         self.assertIn('PO BOX 8102', response[0]['address_street_1'])
         self.assertIn('60680',response[0]['address_zip'])
         # testing for year sensitivity
         self.assertIn('O', response[0]['incumbent_challenge'])
 
-    def test_candidate_committes(self):
-        response = self._results('/candidate/P80003338?year=*')
```

Comment:
I removed this test because it was checking the embedded committees that were in the candidate information.